neard/res/genesis_config.json

```diff
@@ -194,7 +193,7 @@
   "validators": [
     {
       "account_id": "test.near",
-      "public_key": "ed25519:Fr9rU2jBF5dWfXJ9ewH7qAuirJMa2DKJ2QxV4DnizRnh",
+      "public_key": "ed25519:3MSgRAN3847w5qhq13cmYVVExVCshSf6GKFozCrQTnD6",
```

Comment:
🤔

Reply:
Hmm this should be fixed to avoid this file change every time

Reply:
Pinned to seed keypair of test.near.